policyengine_us/parameters/gov/states/ri/tax/income/agi/subtractions/social_security/limit/birth_year.yaml

```diff
@@ -18,3 +21,4 @@ metadata:
 values:
   2021-01-01: 1955
   2022-01-01: 1956
+  2024-01-01: 1958
```

Comment:
Assuming 2023 was 1957?

Reply:
Actually could you run an upstream command? I see the 2023 values in the main code

Reply:
Fixed